### ISSUE TITLE
[jackson-dataformat-csv 2.13.1] Parsing column wrapped in double quotes over multiple lines

### ISSUE THREAD

**Issue by sbmaggarwal** [FasterXML/jackson-dataformats-text#318]
I have the following CSV format:
```
id,name,description,age
23,Anna,"Self-made
Chef
Shoemaker",23
```
The double-quotes are only present if the attribute is multi-line. While I am already been able to read the normal CSV correctly:
```
@Bean
public CsvMapper csvMapper() {
    CsvMapper csvMapper = new CsvMapper();
    csvMapper.registerModule(new JavaTimeModule());
    csvMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
    return csvMapper;
}
```
How do I parse the given format and get the whole `Self-made\n Chef\n Shoemaker` into the description attribute?

**Comment by cowtowncoder** [comment]
Could you express this in a form of unit test? I am not sure I understand this part:

    The double-quotes are only present if the attribute is multi-line.

specifically, is this about reading or writing CSV? What is not working?
If reading, how are you attempting to read the content?

**Comment by sbmaggarwal** [comment]
Actually, the issue by solved by just updating the dependency to `2.14`. Someone suggested the same over my [SO question](https://stackoverflow.com/questions/72844595/parsing-double-quote-with-new-line-from-csv-using-jackson-dataformat-csv).
